lib/t/push_notifications/dispatch_job.ex

```diff
@@ -158,7 +174,6 @@ defmodule T.PushNotifications.DispatchJob do
   defp profile_name(user_id) do
     Accounts.Profile
     |> where(user_id: ^user_id)
-    |> select([p], p.name)
```

Comment:
`profile_name` function name no longer reflects the meaning of the function

Reply:
changed name of function

Reply:
I guess you only take `name` and `gender` from Profile, so probably can select just them here